   <!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="/assets/favicon.ico/favicon.ico">
    <!--<link rel="apple-touch-icon" sizes="57x57" href="/apple-icon-57x57.png">
    <link rel="apple-touch-icon" sizes="60x60" href="/apple-icon-60x60.png">
    <link rel="apple-touch-icon" sizes="72x72" href="/apple-icon-72x72.png">
    <link rel="apple-touch-icon" sizes="76x76" href="/apple-icon-76x76.png">
    <link rel="apple-touch-icon" sizes="114x114" href="/apple-icon-114x114.png">
    <link rel="apple-touch-icon" sizes="120x120" href="/apple-icon-120x120.png">
    <link rel="apple-touch-icon" sizes="144x144" href="/apple-icon-144x144.png">
    <link rel="apple-touch-icon" sizes="152x152" href="/apple-icon-152x152.png">
    <link rel="apple-touch-icon" sizes="180x180" href="/apple-icon-180x180.png">
    <link rel="icon" type="image/png" sizes="192x192"  href="/android-icon-192x192.png">
    <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
    <link rel="icon" type="image/png" sizes="96x96" href="/favicon-96x96.png">
    <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
    <link rel="manifest" href="/manifest.json">
    <meta name="msapplication-TileColor" content="#ffffff">
    <meta name="msapplication-TileImage" content="/ms-icon-144x144.png">
    <meta name="theme-color" content="#ffffff">-->
    <title>Damha Research Group - Latest Research</title>

    <!-- Bootstrap core CSS -->
    <link href="css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom styles for this template -->
    <link href="css/main.css" rel="stylesheet">
</head>

<body>

    <!-- NAVBAR START -->

    <nav class="navbar navbar-default navbar-expand-md navbar-dark fixed-top">
        <a class="navbar-brand" href="index.html">DAMHA LAB</a>
        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarsExampleDefault" aria-controls="navbarsExampleDefault" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
      </button>

        <div class="collapse navbar-collapse" id="navbarsExampleDefault">
            <ul class="navbar-nav mr-auto">
                <li class="nav-item dropdown">
                    <a class="nav-link dropdown-toggle" id="dropdown01" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Our Group</a>
                    <div class="dropdown-menu" aria-labelledby="dropdown01">
                        <a class="dropdown-item" href="damha.html">Prof. Damha</a>
                        <a class="dropdown-item" href="members.html">Group Members</a>
                        <a class="dropdown-item" href="photos.html">Photos</a>
                    </div>
                </li>

                <li class="nav-item">
                    <a class="nav-link" href="research.html">Research</a></li>
                <li class="nav-item">
                    <a class="nav-link" href="publications.html">Publications</a></li>
                <li class="nav-item dropdown">
                    <a class="nav-link active dropdown-toggle" id="dropdown01" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Latest</a>
                    <div class="dropdown-menu" aria-labelledby="dropdown01">
                        <a class="dropdown-item" href="latest.html">Research</a>
                        <a class="dropdown-item" href="news.html">News</a>
                    </div>
                </li>
                <li class="nav-item">
                    <a class="nav-link" href="contact.html">Contact</a></li>

            </ul>

        </div>
    </nav>

    <!-- NAVBAR END -->

    <img id="banner" src="images/banner-new.jpg" alt="bubbles">
    <div class="container">
        <div class="blog-header">
            <div class="container">
                <h2 class="blog-title">LATEST RESEARCH</h2>
                <!--<p class="lead blog-description"></p>-->
            </div>
        </div>
<div class=".lead"></div>
<div class="container">

<div class="row">

<!--BLOG POSTS-->
<div class="col-sm-8 blog-main">

    <!--BLOG POST TEMPLATE    
    <div class="blog-post">
        <h3 class="blog-post-title">Blog Post Template</h3>
        <p class="blog-post-meta">Date</p>

        <div><img src="" alt="" class="blog-image"></div>
        <p>Abstract</p>
        <p>Reference</p>
        <p>View the full article <a href="">here</a>.</p>
        </div>
        <!-- /.blog-post --> 
<!--END OF TEMPLATE -->    
    
    <div class="blog-post">
        <h3 class="blog-post-title">Mapping the Affinity Landscape of Thrombin-Binding Aptamers on 2'F-ANA/DNA Chimeric G-Quadruplex Microarrays</h3>
        <p class="blog-post-meta">February 2017</p>

        <div><img src="images/latest_research/2017-lietard-NAR.jpeg" alt="2017-lietard-NAR" class="blog-image"></div>
        
        <p>In situ fabricated nucleic acids microarrays are versatile and very high-throughput platforms for aptamer optimization and discovery, but the chemical space that can be probed against a given target has largely been confined to DNA, while RNA and non-natural nucleic acid microarrays are still an essentially uncharted territory. 2΄-Fluoroarabinonucleic acid (2΄F-ANA) is a prime candidate for such use in microarrays. Indeed, 2΄F-ANA chemistry is readily amenable to photolithographic microarray synthesis and its potential in high affinity aptamers has been recently discovered. We thus synthesized the first microarrays containing 2΄F-ANA and 2΄F-ANA/DNA chimeric sequences to fully map the binding affinity landscape of the TBA1 thrombin-binding G-quadruplex aptamer containing all 32 768 possible DNA-to-2΄F-ANA mutations. The resulting microarray was screened against thrombin to identify a series of promising 2΄F-ANA-modified aptamer candidates with Kds significantly lower than that of the unmodified control and which were found to adopt highly stable, antiparallel-folded G-quadruplex structures. The solution structure of the TBA1 aptamer modified with 2΄F-ANA at position T3 shows that fluorine substitution preorganizes the dinucleotide loop into the proper conformation for interaction with thrombin. Overall, our work strengthens the potential of 2΄F-ANA in aptamer research and further expands non-genomic applications of nucleic acids microarrays.</p>
        <p>J. Lietard†, H. Abou-Assi, I. Gomez-Pinto, C. Gonzalez, M.M. Somoza and M.J. Damha† (2017) Mapping the affinity landscape of thrombin-binding aptamers on 2'F-ANA/DNA chimeric G-quadruplex microarrays.  Nucleic Acids Research, 45: 1619–1632 , doi: 10.1093/nar/gkw1357</p>
        <p>View the full article <a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkw1357">here</a>.</p>
        </div>
        <!-- /.blog-post -->    

    <div class="blog-post">
        <h3 class="blog-post-title">Fluorescent Branched RNAs for High-Throughput Analysis of Dbr1 Enzyme Kinetics and Inhibition</h3>
        <p class="blog-post-meta">January 2017</p>

        <div><img src="images/latest_research/2017-katolik-acs.jpeg" alt="2017-katolik-acs" class="blog-image"></div>
        
        <p>We have developed fluorescent 2′,5′ branched RNAs (bRNA) that permit real time monitoring of RNA lariat (intron) debranching enzyme (Dbr1) kinetics. These compounds contain fluorescein (FAM) on the 5′ arm of the bRNA that is quenched by a dabcyl moiety on the 2′ arm. Dbr1-mediated hydrolysis of the 2′,5′ linkage induces a large increase in fluorescence, providing a convenient assay for Dbr1 hydrolysis. We show that unlabeled bRNAs with non-native 2′,5′-phosphodiester linkages, such as phosphoramidate or phosphorothioate, can inhibit Dbr1-mediated debranching with IC50 values in the low nanomolar range. In addition to measuring kinetic parameters of the debranching enzyme, these probes can be used for high throughput screening (HTS) of chemical libraries with the aim of identifying Dbr1 inhibitors, compounds that may be useful in treating neurodegenerative diseases and retroviral infections.</p>
        <p>A. Katolik, N.E. Clark†, N. Tago, E.J. Montemayor, P.J. Hart†, and M.J. Damha† (2017) Fluorescent branched RNAs for high-throughput analysis of Dbr1 enzyme kinetics and inhibition.  ACS Chemical Biology, 12: 622–627, doi: 10.1021/acschembio.6b00971</p>
        <p>View the full article <a href="http://pubs.acs.org/doi/abs/10.1021/acschembio.6b00971">here</a>.</p>
        </div>
        <!-- /.blog-post --> 
    
    <div class="blog-post">
        <h3 class="blog-post-title">Stabilization of i-Motif Structures by 2'-B-Fluorination of DNA</h3>
        <p class="blog-post-meta">June 2016</p>

        <div><img src="images/latest_research/2016-abouassi-nar.png" alt="2016-abouassi-nar" class="blog-image"></div>
        
        <p>This publication was recommended on F1000Prime by Jean-Louis Mergny, a Research Director in Structural Biology at the Université de Bordeaux. Congratulations to Hala! </p>

        <p>i-Motifs are four-stranded DNA structures consisting of two parallel DNA duplexes held together by hemi-protonated and intercalated cytosine base pairs (C:CH+). They have attracted considerable research interest for their potential role in gene regulation and their use as pH responsive switches and building blocks in macromolecular assemblies. At neutral and basic pH values, the cytosine bases deprotonate and the structure unfolds into single strands. To avoid this limitation and expand the range of environmental conditions supporting i-motif folding, we replaced the sugar in DNA by 2-deoxy-2-fluoroarabinose. We demonstrate that such a modification significantly stabilizes i-motif formation over a wide pH range, including pH 7. Nuclear magnetic resonance experiments reveal that 2-deoxy-2-fluoroarabinose adopts a C2′-endo conformation, instead of the C3′-endo conformation usually found in unmodified i-motifs. Nevertheless, this substitution does not alter the overall i-motif structure. This conformational change, together with the changes in charge distribution in the sugar caused by the electronegative fluorine atoms, leads to a number of favorable sequential and inter-strand electrostatic interactions. The availability of folded i-motifs at neutral pH will aid investigations into the biological function of i-motifs in vitro, and will expand i-motif applications in nanotechnology.</p>

        <p>H. Abou Assi, R.W. Harkness V, N. Martín-Pintado, C.J. Wilds, R. Campos-Olivas, A.K. Mittermaier, C. González†, and M.J. Damha†​ (2016) Stabilization of i-motif structures by 2'-β-fluorination of DNA , Nucleic Acids Research. doi: 10.1093/nar/gkw402. </p>

        <p>View the full article <a href="https://academic.oup.com/nar/article/44/11/4998/2468324/Stabilization-of-i-motif-structures-by-2">here</a>.</p>

        </div>
        <!-- /.blog-post -->

    <div class="blog-post">
        <h3 class="blog-post-title">Synthesis and Studies of Oxepane Nucleoside Analogues</h3>
        <p class="blog-post-meta">October 2015</p>

        <p>The synthesis of a novel series of seven-membered ring nucleoside analogues as candidates for biological screening and gene silencing applications is described. The key step in the synthetic approach is a stereoselective synthesis of an epoxide that is used as a common synthetic intermediate to prepare functionalized oxepane nucleoside derivatives. The conformational landscape and preferred ring-puckering of selected oxepane nucleosides was also studied by NMR, X-ray crystallography, and quantum mechanical calculations.</p>

        <p>M. Habibian, S. Martinez-Montero, G. Portella, Z. Chua, D.S. Bohle, M. Orozco, and M.J. Damha† (2015) Seven-membered ring nucleoside analogues: stereoselective synthesis and studies on their conformational properties, Organic Letters, 17 (21): 5416-5419.</p>
    
        <p>View the full article <a href="http://pubs.acs.org/doi/abs/10.1021/acs.orglett.5b02769">here</a>.</p>
        </div>
        <!-- /.blog-post -->    
 
    <div class="blog-post">
        <h3 class="blog-post-title">Nucleotide Sugar Pucker Preference Mitigates Excision by HIV-1 RT</h3>
        <p class="blog-post-meta">July 2015</p>

        <div><img src="images/latest_research/2015-yamada-acs.jpeg" alt="2015-yamada-acs" class="blog-image"></div>
        
        <p>A series of DNA primers containing nucleotides with various sugar pucker conformations at the 3′-terminus were chemically synthesized by solid-phase synthesis. The ability of wild-type (WT) HIV-1 reverse transcriptase (RT) and AZT-resistant (AZTr) RT to excise the 3′-terminal nucleotide was assessed. Nucleosides with a preference for the North conformation were more refractory to excision by both WT-RT and AZTr-RT. We found that DNA primers that contain North puckered-nucleotides at the 3′-terminus can also affect the translocation status of the RT/template/primer complex, which provides an underlying mechanism to avoid being excised. Together, these results point to a correlation between the sugar conformation of the 3′-terminal nucleotide, the precise position of HIV-1 RT on its nucleic acid substrate, and, in turn, its catalytic function. Nucleotide sugar conformation is therefore an important parameter in defining the susceptibility to RT-catalyzed phosphorolytic excision.</p>

        <p>K. Yamada, A.S. Wahba, J. Bernatchez, T. Ilina, S. Martinez-Montero, M. Habibian, G.F. Deleavey, M. Gotte, M.A. Parniak, and M.J. Damha† (2015), ACS Chemical Biology</p>

        <p>You can view the full article <a href="http://pubs.acs.org/doi/abs/10.1021/acschembio.5b00263">here</a>.</p>
        </div>
        <!-- /.blog-post -->   
    
    <div class="blog-post">
        <h3 class="blog-post-title">Locked 2'-diF-rU Modified Nucleic Acids</h3>
        <p class="blog-post-meta">June 2015</p>

        <div><img src="images/latest_research/2015-martinez-acs.jpeg" alt="2015-martinez-acs" class="blog-image"></div>
        
        <p>2′-Deoxy-2′,4′-difluorouridine (2′,4′-diF-rU) was readily incorporated into DNA and RNA oligonucleotides via standard solid phase synthesis protocols. NMR and thermal denaturation (Tm) data of duplexes was consistent with the 2′,4′-diF-rU nucleotides adopting a rigid North (RNA-like) sugar conformation, as previously observed for the nucleoside monomer. The impact of this modification on Tm is neutral when incorporated within RNA:RNA duplexes, mildly destabilizing when located in the RNA strand of a DNA:RNA duplex, and highly destabilizing when inserted in the DNA strand of DNA:RNA and DNA:DNA duplexes. Molecular dynamics calculations suggest that the destabilization effect in DNA:DNA and DNA:RNA duplexes is the result of structural distortions created by A/B junctions within the helical structures. Quantum mechanics calculations suggest that the “neutral” effect imparted to A-form duplexes is caused by alterations in charge distribution that compensate the stabilizing effect expected for a pure North-puckered furanose sugar. 2′,4′-diF-RNA modified siRNAs were able to trigger RNA interference with excellent efficiency. Of note, incorporation of a few 2′,4′-diF-rU residues in the middle of the guide (antisense) strand afforded siRNAs that were more potent than the corresponding siRNAs containing LNA and 2′-F-ANA modifications, and as active as the 2′-F-RNA modified siRNAs.

        <p>S. Martinez-Montero, G.F. Deleavey, N. Martin-Pintado, J. Fakhoury, C. Gonzalez, and Masad J. Damha† (2015) ACS Chemical Biology</p>

        <p>You can view the full article <a href="http://pubs.acs.org/doi/abs/10.1021/acschembio.5b00218">here</a>.</p>
        </div>
        <!-- /.blog-post -->  

    
    <div class="blog-post">
        <h3 class="blog-post-title">Synthesis and Properties of 2',4'-diFANA Modified Nucleic Acids</h3>
        <p class="blog-post-meta">March 2015</p>

        <div><img src="images/latest_research/2015-martinez-joc.jpeg" alt="2015-martinez-joc" class="blog-image"></div>
        
        <p>We report the synthesis, thermal stability, and RNase H substrate activity of 2′-deoxy-2′,4′-difluoroarabino-modified nucleic acids. 2′-Deoxy-2′,4′-difluoroarabinouridine (2,′4′-diF-araU) was prepared in a stereoselective way in six steps from 2′-deoxy-2′-fluoroarabinouridine (2′-F-araU). NMR analysis and quantum mechanical calculations at the nucleoside level reveal that introduction of 4′-fluorine introduces a strong bias toward the North conformation, despite the presence of the 2′-βF, which generally steers the sugar pucker toward the South/East conformation. Incorporation of the novel monomer into DNA results on a neutral to slightly stabilizing thermal effect on DNA–RNA hybrids. Insertion of 2′,4′-diF-araU nucleotides in the DNA strand of a DNA–RNA hybrid decreases the rate of both human and HIV reverse transcriptase-associated RNase H-mediated cleavage of the complement RNA strand compared to that for an all-DNA strand or a DNA strand containing the corresponding 2′-F-araU nucleotide units, consistent with the notion that a 4′-fluorine in 2′-F-araU switches the preferred sugar conformation from DNA-like (South/East) to RNA-like (North).</p>

        <p>S. Martinez-Montero, G. Deleavey, A. Dierker-Viik, P. Lindovska, T. Ilina, G. Portella, M. Orozco, M.A. Parniak, C. Gonzalez, and M.J. Damha† (2015), Journal of Organic Chemistry, 80 (6): 3083-3091.</p>
        <p>You can view the full article <a href="http://pubs.acs.org/doi/abs/10.1021/jo502948t">here</a>.</p>
        </div>
        <!-- /.blog-post -->   
 
  <div class="blog-post">
        <h3 class="blog-post-title">Synthesis, Structure, and Conformational Analysis of Six-Membered 1,3-Oxathiane Nucleoside Analogues</h3>
        <p class="blog-post-meta">February 2015</p>

    <div><img src="images/latest_research/2015-abouassi-eurjoc.jpg" alt="2015-abouassi-eurjoc" class="blog-image"></div>
      
      <p>We report on the synthesis, characterization, and conformational analysis of 1,3-oxathiane nucleosides of both pyrimidine and purine nucleobases. The novel nucleoside analogues were prepared from readily available starting materials as α/β anomeric mixtures. The purine nucleosides were obtained as separable N7/N9 regioisomers. The structures were identified by extensive NMR analysis. The X-ray structure of an N7-β-OMe-purine nucleoside analogue shows that the 1,3-oxathiane sugar ring adopts a perfect chair conformation with both the hydroxymethyl and the nucleobase placed in equatorial positions.</p>    

    <p>Abou Assi, H.; Martinez-Montero, S.; Dixit, D.; Chua, Z.; Bohle, D.S.; Damha, M.J. (2015). Eur. J. Org. Chem. 1945-1953.</p>
      
    <p>View the full article <a href="http://onlinelibrary.wiley.com/doi/10.1002/ejoc.201403642/abstract">here</a>.</p>
        </div>
        <!-- /.blog-post -->      
 
    <div class="blog-post">
        <h3 class="blog-post-title">NAR Breakthrough! Structural Basis of Lariat RNA Recognition by the Intron Debranching Enzyme Dbr1</h3>
        <p class="blog-post-meta">July 2014</p>

        <div><img src="images/latest_research/2014-katolik-nar.jpg" alt="2014-katolik-nar" class="blog-image"></div>
        
        <p>The enzymatic processing of cellular RNA molecules requires selective recognition of unique chemical and topological features. The unusual 2',5'-phosphodiester linkages in RNA lariats produced by the spliceosome must be hydrolyzed by the intron debranching enzyme (Dbr1) before they can be metabolized or processed into essential cellular factors, such as snoRNA and miRNA. Dbr1 is also involved in the propagation of retrotransposons and retroviruses, although the precise role played by the enzyme in these processes is poorly understood. Here, we report the first structures of Dbr1 alone and in complex with several synthetic RNA compounds that mimic the branchpoint in lariat RNA. The structures, together with functional data on Dbr1 variants, reveal the molecular basis for 2',5'-phosphodiester recognition and explain why the enzyme lacks activity toward 3',5'-phosphodiester linkages. The findings illuminate structure/function relationships in a unique enzyme that is central to eukaryotic RNA metabolism and set the stage for the rational design of inhibitors that may represent novel therapeutic agents to treat retroviral infections and neurodegenerative disease.</p>
        <p>E.J. Montemayor, A. Katolik, A.B. Taylor, J. Scheurmann, D.J. Combs, R. Johnsson, S.P. Holloway, S.W. Stevens†, M.J. Damha†,  and P.J. Hart† (2014) Structural basis of lariat RNA recognition by the intron debranching enzyme Dbr1, Nucleic Acids Research ("Breakthrough Article")</p>
        <p>View the full article <a href="https://academic.oup.com/nar/article/42/16/10845/2902900/Structural-basis-of-lariat-RNA-recognition-by-the">here</a>.</p>
        </div>
        <!-- /.blog-post -->  
    
                  
<!--UNCOMMENT FOR PAGINATION
    <nav class="blog-pagination">
    <a class="btn btn-outline-primary" href="#">Older</a>
    <a class="btn btn-outline-secondary disabled" href="#">Newer</a>
</nav>
END OF PAGINATION -->
    
</div>
<!-- /.blog-main -->

        <div class="col-sm-3 offset-sm-1 blog-sidebar">
            <div class="sidebar-module sidebar-module-inset">
                <h4>About</h4>
                <p>Abstracts of peer-reviewed articles recently published by the Damha Research Group.</p>
            </div>
            <!--<div class="sidebar-module">
                <h4>Archives</h4>
                <ol class="list-unstyled">
                    <li><a href="#">March 2014</a></li>
                    <li><a href="#">February 2014</a></li>
                    <li><a href="#">January 2014</a></li>
                    <li><a href="#">December 2013</a></li>
                    <li><a href="#">November 2013</a></li>
                    <li><a href="#">October 2013</a></li>
                    <li><a href="#">September 2013</a></li>
                    <li><a href="#">August 2013</a></li>
                    <li><a href="#">July 2013</a></li>
                    <li><a href="#">June 2013</a></li>
                    <li><a href="#">May 2013</a></li>
                    <li><a href="#">April 2013</a></li>
                </ol>
            </div>-->

        </div>
        <!-- /.blog-sidebar -->

    </div>
    <!-- /.row -->

</div>
<!-- /.container -->

<footer><hr>COPYRIGHT MASAD J. DAMHA. DESIGNED & MAINTAINED BY <a href="mailto:danielle.vlaho@mail.mcgill.ca">DANIELLE VLAHO.</a>
</footer>
        
  <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
    <script>
        window.jQuery || document.write('<script src="assets/js/vendor/jquery.min.js"><\/script>')
    </script>
    <script src="assets/js/vendor/popper.min.js"></script>
    <script src="dist/js/bootstrap.min.js"></script>
    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
    <script src="assets/js/ie10-viewport-bug-workaround.js"></script>
</body>
</html>